app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt

```diff
@@ -163,6 +165,7 @@ class MapsFragment : androidx.fragment.app.Fragment(), OnClickListener, OnMapRea
                             findNavController().navigate(
                                 MapsFragmentDirections.actionGlobalLoginFlowGraph())
                         } else {
+                            vm.turnOnTreeCaptureMode()
```

Comment:
This is the change I feel is kind of weird. Invoking `turnOnTreeCaptureMode` on `locationDataCapturer` is much more clearer in terms of the flow than invoking it in the view model (which in turn invokes the method with the same name on `locationDataCapturer`). But I understand mvvm pattern recommends no direct interaction with other types of models on the UI code.